### PR TITLE
feat(queue loader): add curation queue loader script

### PR DESCRIPTION
**Pull request by kkyeboah**
## Goal

Add a script that will move curated items from `curated_feed_queued_items` to the `curated_feed_items` table with the appropriate `time_live`.

## References

JIRA ticket: INFRA-430

### CODE REVIEW

**Review by kschelonka — APPROVED** [review]
This looks good as discussed; would probably be good to add additional documentation around how it must be run (must drain entire queue before next scheduled cron job at 45 on the hour), what web repo logic this is replicating (maybe just a link to the job code on GH), and a quick description of why we are doing this.

File: bin/load-queued-items.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  '                  END AS time_weight,\n' +
+  '                  q.time_added\n' +
+  '                FROM `readitla_ril-tmp`.curated_feed_queued_items q\n' +
+  '                WHERE q.feed_id = :feedId\n' +
+  "                AND q.status = 'ready'\n" +
+  '                ORDER BY q.weight DESC, time_weight DESC, q.time_added ASC\n' +
+  '                LIMIT 1;';
+
+async function getNextQueuedItem(feedId) {
+  const dayWeight = getRandomInt(1, 101);
```
Comment (sri-kirsh) [inline bin/load-queued-items.ts:54]: thinking why order matters here as the day and week weights are all assigned randomly? 

to confirm: we just want assign all items in the queue and we are not prioritizing them based on if its `relevant length` is `day` or `week`

**Review by sri-kirsh — APPROVED** [review]

File: bin/load-queued-items.ts (hunk 1 above)
Comment (kkyeboah) [reply bin/load-queued-items.ts:54]: replicating the logic in https://github.com/Pocket/Web/blob/main/processes/curation/add_feed_items.py

**Review by kschelonka — APPROVED** [review]
🎉

**Pull request merged**